metasrv/tests/it/main.rs

```diff
@@ -12,6 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Because the compiler complains about recursion limit for a trait requirement check...
```

Comment:
There should be more comments or a reference link here to explain this case.

Reply:
Yes sir~